README.md

```diff
@@ -13,8 +13,14 @@ This project replaces the legacy "VIVO Installer". It is used to create a "third
    home/src/main/resources/config/runtime.properties
    home/src/main/resources/rdf/display/firsttime/menu.n3
    ```
-4. Build the custom webapp
+4. Build base VIVO webapp
    ```
+   cd <location of VIVO source code>
+   mvn clean install
```

Comment:
As discussed in Slack, this step is only necessary during the PR-testing phase. Once the following PRs are merged and either released to the [Sonatype snapshot](https://oss.sonatype.org/content/repositories/snapshots/org/vivoweb/) repository or [Maven Central](https://search.maven.org/search?q=org.vivoweb), the step will no longer be need.
- https://github.com/vivo-project/VIVO/pull/197
- https://github.com/vivo-project/Vitro/pull/192

That said, I am inclined to not include the changes in this PR as a part of the README... although having the step in the PR instructions would be helpful... unless you are the last tester before merge.